spec/facets/ColumnLineageDatasetFacet.json

```
@@ -32,6 +32,32 @@
                         "field": {
                           "type": "string",
                           "description": "The input field"
+                        },
+                        "transformations": {
+                          "type": "array",
+                          "items": {
+                            "type": "object",
+                            "properties": {
+                              "type": {
+                                "description": "The type of the transformation. Allowed values are: IDENTITY, TRANSFORMED, INDIRECT",
+                                "type": "string"
+                              },
+                              "subtype": {
+                                "type": "string",
+                                "description": "The subtype of the transformation"
+                              },
+                              "description": {
+                                "type": "string",
+                                "description": "a string representation of the transformation applied"
+                              },
+                              "masking": {
+                                "type": "boolean",
+                                "description": "is transformation masking the data or not"
+                              }
+                            }
+                          },
+                          "additionalProperties": true,
+                          "required": ["type"]
```

```suggestion
                            },
                            "required": ["type"],
                            "additionalProperties": true
                          }
```

I think you meant to require `type` in nested object in array.

---

thanks for spotting that! great eye for details 🥇